angular/demo/bootstrap/src/app/samples/accordion/playground.route.ts

```diff
@@ -22,13 +21,13 @@ const undefinedAccordionConfig = getUndefinedValues(getAccordionDefaultConfig())
 	</div>`,
 })
 export default class PlaygroundComponent {
-	@ViewChild(AccordionDirective) widget!: AccordionDirective;
-	@ViewChildren(AccordionItemComponent) itemWidgets!: QueryList<AccordionItemComponent>;
+	readonly widget = viewChild.required(AccordionDirective);
+	readonly itemWidgets = viewChildren(AccordionItemComponent);
```

Comment:
should not be required judging by the previous use?

Reply:
viewChildren does not have a `required`